docs/guides/security.md

```diff
@@ -0,0 +1,49 @@
+# Security and Privacy
+
+One of our main pillars is the Security and Privacy of our customers information. This section highlights some of the specific security measures that we have taken to achieve this objective.
+
+## Document encryption and storage
+
+Protecting the confidentiality, integrity, and availability of our customers' data is a critical priority for us. As part of our commitment to security, we have implemented a strict policy for the storage of documents that requires all documents to be stored solely in an encrypted format and on servers that are certified by ISO 27001, within the Azure cloud computing platform.
```

Comment:
@prisas This is could be confusing for the user.  We are pushing for client side E2E encryption and therefore we could store our data on a public NAS and it should not matter to them.  Now we are saying that storing on Azure brings security benefits because it is encrypted at rest and ISO 27001.  While the latter is true and important for our customer, we should stress in the section that document are encrypted client side and that Azure security is just an additional safety.  Othersie it dilutes our USP.